spec/features/content_change_requests_spec.rb

```diff
@@ -73,14 +85,17 @@
   def user_makes_a_content_change_request(details)
     visit "/"
 
-    click_on "Content changes and new content requests"
+    click_on "Request a content change or new content on GOV.UK"
 
     expect(page).to have_content("You'll get an automated response to confirm we've received your request. We'll then review your request within 2 working days.")
 
     fill_in "Title of request", with: details[:title] unless details[:title].nil?
-    fill_in "Details of the requested change", with: details[:details_of_change]
-    fill_in "URL", with: details[:url]
-    fill_in "Does this affect any other URLs (including any existing Welsh translations that need to be updated)? Put each new URL on a new line.", with: details[:related_urls]
+    select details[:reason_for_request], from: "What’s the reason for the request?" unless details[:reason_for_request].nil?
```

Comment:
This should be updated if it's meant to be "change" rather than "request"

Reply:
done